src/utilities/getShortcut.ts

```diff
@@ -0,0 +1,29 @@
+import { registerCommand, executeCommand } from "../utilities/getRandom";
+import { SHORT_CUT } from "../const";
+import * as os from "os";
+
+export function getRandomShortcut() {
+  registerCommand("getRandomShortcut", SHORT_CUT);
+  const shortcut = executeCommand("getRandomShortcut");
```

Comment:
ここって単純に１行で書けそうにないですかね？👀
```suggestion
  const shortcut = SHORT_CUT[Math.floor(Math.random() * SHORT_CUT.length)];
```
`registerCommand` や `executeCommand` を作られた意図が知りたいかもです🙏